app/controllers/dashboard/users_controller.rb

```diff
@@ -16,4 +19,16 @@ def resources_path
   def show_location(resource)
     dashboard_user_path(resource)
   end
+
+  def respond_with_updated_resource
```

Comment:
Ah i see. This is because we are using he users controller. Maybe we should have another controller to handle this. E.g. LocalesController which handles the update method